x/deployment/keeper/keeper_test.go

```diff
@@ -68,6 +68,19 @@ func Test_Create(t *testing.T) {
 		_, ok := keeper.GetGroup(ctx, testutil.GroupID(t))
 		assert.False(t, ok)
 	})
+
+	t.Run("close the first Group", func(t *testing.T) {
+		if len(groups) > 0 {
```

Comment:
this is always true.

do a `require.NotEmpty(t,groups)` if you really want to for clarity, it'd the same as the `if {} else {}` logic here.

Reply:
Actually, I think that since you have `Test_CloseGroup` below, none of these need to be here - leave `Test_Create` as it is.